CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 # Changelog
 All notable changes to FlureeDB will be documented in this file.
 
-## 1.0.0
+## 1.0.0-rc15
```

Comment:
1.0.0-rc15 is already released w/o this change. Should we move this to a newer section?